### ISSUE TITLE
Install fails on Azure / Windows

### ISSUE THREAD

**Issue by JedWatson** [dachev/node-cld#22]
Hi,

@shauntrennery recently opened an issue on [one of my projects](https://github.com/keystonejs/keystone-utils) that uses cld (via [limax](https://github.com/lovell/limax) by @lovell): keystonejs/keystone-utils#15

Just wanted to see if you can provide any insights into why this would be happening, or how we can fix it.

The error:

```
Application has thrown an uncaught exception and is terminated:
Error: Cannot find module './build/Release/cld'
at Function.Module._resolveFilename (module.js:336:15)
at Function.Module._load (module.js:278:25)
at Module.require (module.js:365:17)
at require (module.js:384:17)
at Object. (D:\home\site\wwwroot\node_modules\keystone-utils\node_modules\limax\node_modules\cld\index.js:2:12)
at Module._compile (module.js:460:26)
at Object.Module._extensions..js (module.js:478:10)
at Module.load (module.js:355:32)
at Function.Module._load (module.js:310:12)
at Module.require (module.js:365:17)
```

**Comment by lovell** [comment]
I've published v1.1.1 of `limax` to npm with an update to the latest `cld` as it has recent commits/fixes related to Windows problems.

This seems to work for me on a Window's laptop - I'll add a AppVeyor CI job to `limax` to test Azure also.

**Comment by lovell** [comment]
Azure-based Appveyor CI also working - see https://ci.appveyor.com/project/lovell/limax

**Comment by sachss** [comment]
The inclusion of Limax and specifically the CLD dependency makes it so that we can no longer deploy on a 2GiB EC2 instance.  **Can this be removed until it can be implemented as an optional feature?**  

This issue has been closed as a memory issue by node-cld: [Issue #19](https://github.com/dachev/node-cld/issues/19) 

More discussion of this issue and confirmation of the memory error is discussed on Keystone at [Issue #1476](https://github.com/keystonejs/keystone/issues/1476)

**Comment by chuckwilliams37** [comment]
@JedWatson I was receiving that error until I completely deleted my project node_modules, then installed 'make' and re-ran 'npm install'.
